### ISSUE TITLE
JAWS announces "password edit" multiple times when navigating line-by-line in IE11 

### ISSUE THREAD

**Issue by bethwhitmer** [FreedomScientific/VFO-standards-support#443]
### Summary

In IE11, when the user is navigating line-by-line using the down arrow, JAWS will announce "password edit" multiple times for an <input type="password"> control. In Firefox the announcement also seems to be multiplicative. Note that this ONLY happens when using the arrow keys to navigate to the control; it does not happen when tabbing.

1. Navigate to the following Codepen page: https://cdpn.io/jlgauvreau/debug/e51987c66723149bf2439053aa58a77b
2. Use the down arrow to navigate to the "Enter your password" field with JAWS running. 

### Expected result

JAWS should not announce "password edit" multiple times. 

### Actual result

JAWS announces the following in IE11:

Enter your password:
 password edit     password edit  
Enter your password:  password edit     
Type in text.

### Example
Codepen page: https://cdpn.io/jlgauvreau/debug/e51987c66723149bf2439053aa58a77b

#### JAWS version and build number
2020.2008.24
 
#### Operating System and version
Windows 10

#### Browser and version:
IE11, 11.1282.17763.0

**Comment by menovak** [comment]
Tested in JAWS 2020.2003 and NVDA 2020.2 in IE11

JAWS Forms mode (TAB to Enter your password:) announces -> 
Enter your password: password edit  

JAWS Browse (down arrow to Enter your password: ) announces ->
Enter your password:
password edit   password edit 
Enter your password: password edit  
Type in text.

Note -> REPEATED BETH WHITMER's comments

----
NVDA Forms mode (TAB to Enter your password:) announces -> 
Enter your password:  edit  protected  blank

NVDA Browse (down arrow to Enter your password: ) announces ->
Enter your password:  Enter your password:  edit  protected  

-------

Found on:  https://a11ysupport.io/tests/tech__html__input__input-password#assertion-html-input(type-password)_element-convey_value--jaws-ie

That both JAWS and NVDA have some issues with input[type="password"] element.  Not sure if this is related or not.

**Comment by JAWS-test** [comment]
I have this problem with IE 11, Firefox and Chrome (with Chrome only if the focus is already in the field). The reason seems to be the automatic switch to forms mode. If the automatic switch to forms mode is deactivated, multiple output does not take place in the three browsers

**Comment by menovak** [comment]
This issue was previously reported :  ID 113026

Updated the information in BugZilla to include the details provided in Issue #443.
